### PR TITLE
[npo] Add www.2doc.nl (formarly www.hollanddoc.nl) support to npo.py

### PR DESCRIPTION
**Pull request by pim89**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [x] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Explanation of your *pull request* in arbitrary form goes here. Please make sure the description explains the purpose and effect of your *pull request* and is worded well enough to be understood. Provide as much context and examples as possible.

[npo] add www.2doc.nl (formarly www.hollanddoc.nl) support to npo.py. It downloads all videos present on the page of the requested documentary.

### CODE REVIEW

File: youtube_dl/extractor/lego.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+                    })
+        self._sort_formats(formats)
+
+        return {
+            'id': video_id,
+            'title': title,
+            'thumbnail': video_data.get('CoverImageUrl'),
+            'duration': int_or_none(video_data.get('Length')),
+            'formats': formats,
+        }
```
Comment (dstftw) [inline youtube_dl/extractor/lego.py]: This change is irrelevant to PR.

Comment (pim89) [inline youtube_dl/extractor/lego.py]: I've removed the lego commits that appeared when I synced my fork.
I'm not exactly sure how te keep my fork synced but only keep the actual differences (the parts I add) in the pull request. Sorry for this.

File: youtube_dl/extractor/npo.py [hunk 2]
```diff
@@ -526,3 +526,29 @@ class AndereTijdenIE(NPOPlaylistBaseIE):
         },
         'playlist_count': 3,
     }]
+
+
+class TweeDocIE(NPOPlaylistBaseIE):
+    IE_NAME = 'tweedoc'
+    _VALID_URL = 'https?://(?:www\.)2doc\.nl[^.]*/(?P<id>[^/]+)\.html'
+    _PLAYLIST_TITLE_RE = r'(?s)<h1[^>]+class=["\'].*?\bpage-title\b.*?["\'][^>]*>(.+?)</h1>'
```
Comment (dstftw) [inline youtube_dl/extractor/npo.py]: This does not match anything.

File: youtube_dl/extractor/npo.py [hunk 3]
```diff
@@ -526,3 +526,29 @@ class AndereTijdenIE(NPOPlaylistBaseIE):
         },
         'playlist_count': 3,
     }]
+
+
+class TweeDocIE(NPOPlaylistBaseIE):
+    IE_NAME = 'tweedoc'
+    _VALID_URL = 'https?://(?:www\.)2doc\.nl[^.]*/(?P<id>[^/]+)\.html'
```
Comment (dstftw) [inline youtube_dl/extractor/npo.py]: This allows URLs like 2doc.nlabc/... that is incorrect.

**Pull request closed without merging**